tests/nlu/extractors/test_extractor.py

```diff
@@ -105,15 +132,116 @@
                 }
             ],
         ),
+        (
+            "Aarhus",
+            [Token("Aa", 0, 2), Token("r", 2, 3), Token("hu", 3, 5), Token("s", 5, 6)],
+            [
+                {
+                    "entity": "city",
+                    "start": 0,
+                    "end": 2,
+                    "confidence": 0.32,
+                    "value": "Aa",
+                }
+            ],
+            True,
+            [
+                {
+                    "entity": "city",
+                    "start": 0,
+                    "end": 6,
+                    "confidence": 0.32,
+                    "value": "Aarhus",
+                }
+            ],
+        ),
+        (
+            "Aarhus",
+            [Token("Aa", 0, 2), Token("r", 2, 3), Token("hu", 3, 5), Token("s", 5, 6)],
+            [
+                {
+                    "entity": "city",
+                    "start": 0,
+                    "end": 2,
+                    "confidence": 0.32,
```

Comment:
what about the case where there are multiple tokens with different confidences? can you add a test for that as well?

Reply:
test already exists